src/js/stores/QueueStore.js

```diff
@@ -0,0 +1,60 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+
+var AppDispatcher = require("../AppDispatcher");
+var QueueEvents = require("../events/QueueEvents");
+var queueScheme = require("./queueScheme");
+
+function processQueue(queue = []) {
+  return lazy(queue).map(function (entry) {
```

Comment:
Good point! I think that was laziness.. :)
I will look through the code and see where it makes sense and where not.